packages/components/src/MultiSelect/MultiSelect.spec.tsx

```diff
@@ -42,6 +42,20 @@ const MultiSelectWrapper = ({
 const user = userEvent.setup()
 
 describe("<MultiSelect />", () => {
+  describe("accessible name and description", () => {
+    it("has an accessible name and description when provided a description", async () => {
```

Comment:
Nit (kinda annoying that I don't see everything at once :s)
```suggestion
    it("has an accessible name and description when provided a description", () => {
```